knowledge-base/create-custom-jpeg-image-converter-net-standard.md

```diff
@@ -0,0 +1,81 @@
+---
+title: Create Custom JpegImageConverter in .Net Standard
+description: Describes how to create a custom implementation of JpegImageConverterBase in .Net Standard
+type: how-to
+page_title: Create custom implementation of JpegImageConverterBase in .Net Standard
+slug: create-custom-jpeg-image-converter-net-standard
+position: 0
+tags: image, converter, jpeg, jpegimageconverter, netstandard
+res_type: kb
+---
+
+<table>
+<thead>
+	<tr>
+		<th>Product Version</th>
+		<th>Product</th>
+		<th>Author</th>
+	</tr>
+</thead>
+<tbody>
+	<tr>
+		<td>2020.1.506</td>
```

Comment:
The version looks weird. Is this the correct version?

Reply:
I copy-pasted it from the DLL's properties. It is the DLL from the 6th of May. Should it be from a specific LIB or Release?